runner/tests/integration.test.ts

```diff
@@ -149,6 +148,37 @@ describe('Indexer integration', () => {
     `);
 
     expect(logs.length).toEqual(4);
+
+    const { morgs_near_test___logs: _logs }: any = await graphqlClient.request(gql`
```

Comment:
```suggestion
    const { morgs_near_test___logs: logs }: any = await graphqlClient.request(gql`
```
The prefix here is really necessary, it's only really important at the DB level

Reply:
I didn't use this naming due to the above code 
`const { indexer_log_entries: logs }: any = await graphqlClient.request(gql`
what I will do is rename this to `old_logs`. so this can be `logs` as suggested and remove old_logs as apart of the removal PR